### PR TITLE
zh: add single page to describe enterprise edition deployment

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? (Required)

According to some users it's hard to find how to deploy tidb cluster enterprise edition through tidb-operator. So I add a single document to describe how to deploy enterprise edition tidb cluster.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: zh/deploy-tidb-enterprise-edition.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc-enterprise
+```
+
+如果是部署全新集群，运行 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` apply 上述配置即可部署 TiDB 企业版集群及企业版周边工具。
+
+如果是需要将已有集群切换为企业版，只需将已有集群的配置文件按上述格式在 `baseImage` 后添加 "-enterprise" 后缀并使用 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` 更新集群配置即可。
+
+tidb-operator 会自动通过滚动升级的方式将集群镜像更新为企业版镜像。
```
Comment (ran-huang) [inline zh/deploy-tidb-enterprise-edition.md:45]:
```suggestion
TiDB Operator 会自动通过滚动升级的方式将集群镜像更新为企业版镜像。
```

File: zh/deploy-tidb-enterprise-edition.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    baseImage: pingcap/tiflash
+  ...
+  pump:
+    baseImage: pingcap/tidb-binlog
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc
+```
+
+使用 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` 更新集群配置后，tidb-operator 会自动通过滚动升级的方式将集群镜像切换为社区版镜像。
```
Comment (ran-huang) [inline zh/deploy-tidb-enterprise-edition.md:74]:
```suggestion
使用 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` 更新集群配置后，TiDB Operator 会自动通过滚动升级的方式将集群镜像切换为社区版镜像。
```

File: zh/deploy-tidb-enterprise-edition.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+title: 在 Kubernetes 上部署 TiDB 企业版
+summary: 了解如何在 Kubernetes 上部署 TiDB 企业版。
+---
+
+# 部署 TiDB 企业版
+
+本文档介绍如何在 Kubernetes 上部署 TiDB 集群企业版及相应的企业版工具。
```
Comment (ran-huang) [inline zh/deploy-tidb-enterprise-edition.md:8]: Can we add a few words to explain what is the enterprise version?

Comment (lichunzhu) [reply zh/deploy-tidb-enterprise-edition.md]: https://pingcap.com/download-cn/
Use the description from here.

**Comment by ran-huang** [comment]
LGTM

File: zh/deploy-tidb-enterprise-edition.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    baseImage: pingcap/tiflash-enterprise
+  ...
+  pump:
+    baseImage: pingcap/tidb-binlog-enterprise
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc-enterprise
+```
+
+如果是部署全新集群，运行 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` apply 上述配置即可部署 TiDB 企业版集群及企业版周边工具。
```
Comment (DanielZhangQD) [inline zh/deploy-tidb-enterprise-edition.md:45]:
```suggestion
如果是部署全新集群，请参阅[在 Kubernetes 中配置 TiDB 集群](configure-a-tidb-cluster.md) 配置 `tidb-cluster.yaml`，并按上述描述配置企业版镜像，运行 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` 即可部署 TiDB 企业版集群及企业版周边工具。
```

File: zh/deploy-tidb-enterprise-edition.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+如果是需要将已有集群切换为企业版，只需将已有集群的配置文件按上述格式在 `baseImage` 后添加 "-enterprise" 后缀并使用 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` 更新集群配置即可。
```
Comment (DanielZhangQD) [inline zh/deploy-tidb-enterprise-edition.md:47]:
```suggestion
如果是需要将已有集群切换为企业版：

* 将已有集群的配置文件按上述格式在 `baseImage` 后添加 "-enterprise" 后缀并使用 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` 更新集群配置。
* 通过 `kubectl edit tc ${name} -n ${namespace}` 按上述格式为各组件 `baseImage` 后添加 "-enterprise" 后缀，更新集群配置。
```

File: zh/deploy-tidb-enterprise-edition.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    baseImage: pingcap/tiflash
+  ...
+  pump:
+    baseImage: pingcap/tidb-binlog
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc
+```
+
+使用 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` 更新集群配置后，TiDB Operator 会自动通过滚动升级的方式将集群镜像切换为社区版镜像。
```
Comment (DanielZhangQD) [inline zh/deploy-tidb-enterprise-edition.md:78]: Please also add a way with `kubectl edit`, sometimes the original yaml files will be lost.

File: zh/deploy-tidb-enterprise-edition.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+* 方式二：通过 `kubectl edit tc ${name} -n ${namespace}` 按上述格式为各组件 `baseImage` 后添加 "-enterprise" 后缀，更新集群配置。
+
+TiDB Operator 会自动通过滚动升级的方式将集群镜像更新为企业版镜像。
+
+## 切换回社区版本
+
+如果需要将已有集群切换回社区版：
+
+* 方式一：将已有集群的配置文件按上述格式在 `baseImage` 项去除 "-enterprise" 后缀并使用 `kubectl apply -f tidb-cluster.yaml -n ${namespace}` 更新集群配置。
+* 方式二：通过 `kubectl edit tc ${name} -n ${namespace}` 按上述格式为各组件 `baseImage` 后去除 "-enterprise" 后缀，更新集群配置。
```
Comment (DanielZhangQD) [inline zh/deploy-tidb-enterprise-edition.md:59]: 这是要放 L83？`上述`格式在下面

**Pull request merged**